doc/sphinxext/math_symbol_table.py

```diff
@@ -105,7 +104,9 @@ def render_symbol(sym):
 
     lines = []
     for category, columns, syms in symbols:
-        syms = sorted(list(syms))
+        # Assume that lists are correctly sorted
+        if isinstance(syms, set):
+            syms = sorted(list(syms))
```

Comment:
```suggestion
            syms = sorted(syms)
```

sets are iterable so `sorted` can take them directly.

Super picky, take it or leave it.